### PR TITLE
fix: use flashbar to ask if you want to open the app

### PR DESCRIPTION
**Pull request by raredeveloper**

### CODE REVIEW

File: app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt [hunk 1]
```diff
@@ -201,9 +205,18 @@ abstract class BaseEditorActivity :
       return
     }
 
-    Snackbar.make(binding.realContainer, string.msg_action_open_application, Snackbar.LENGTH_LONG)
-      .setAction(string.yes) { IntentUtils.launchApp(this, packageName) }
-      .show()
+    flashbarBuilder(duration = DURATION_INDEFINITE)
+      .successIcon()
```
Comment (itsaky) [inline app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt:209]: Shouldn't this use `infoIcon()`?

**Review by itsaky — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt (hunk 1 above)
Comment (raredeveloper) [reply app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt:209]: true, I'll change it

File: app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt [hunk 2]
```diff
@@ -201,9 +205,18 @@ abstract class BaseEditorActivity :
       return
     }
 
-    Snackbar.make(binding.realContainer, string.msg_action_open_application, Snackbar.LENGTH_LONG)
-      .setAction(string.yes) { IntentUtils.launchApp(this, packageName) }
-      .show()
+    flashbarBuilder(duration = DURATION_INDEFINITE)
```
Comment (raredeveloper) [inline app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt:208]: @itsaky, do I leave the duration undefined and create a variable to check if the Flashbar is already visible, or do I leave the Flashbar with a short duration?

Comment (itsaky) [reply app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt:208]: >  do I leave the duration undefined

I think, yes. `DURATION_INDEFINITE` should be fine.

> and create a variable to check if the Flashbar is already visible

If you plan to dismiss the Flashbar if it is already visible, you'll probably keep a reference to the shown Flashbar. In that case, please make sure that you release the Flashbar instance properly.

Comment (raredeveloper) [reply app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt:208]: Okay

**Pull request closed without merging**